packages/terra-site/src/App.jsx

```diff
@@ -15,6 +15,8 @@ import FormComponentNavigation from './examples/form/FormComponentNavigation';
 import ListComponentNavigation from './examples/list/ListComponentNavigation';
 import IconComponentNavigation from './examples/icon/IconComponentNavigation';
 
+import CernerConsumerTheme from 'cerner-consumer-theme/scoped-theme';
```

Comment:
Let's remove this, we can add it back in a separate PR for adding the consumer theme.

Reply:
Whoops. Didn't mean to check that in. https://github.com/cerner/terra-core/pull/966/commits/10cf489704028579deffb25cd49febe52e742b82